### PR TITLE
feat: new library Golan gophers

### PR DESCRIPTION
**Pull request by excalibot**
Golang gophers we all love to spice up your drawings.

Submitted by @github.com/gregoryv

Items: gopher front, gopher up, gopher right, gopher angle, gopher left, gopher down

[Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fgregory-golang-gophers-1667862212999%2Flibraries%2Fgregory%2Fgolang-gophers.excalidrawlib%3Fraw%3Dtrue)

### CODE REVIEW

**Comment by gregoryv** [comment]
The name should be "Golang gophers", misspelled "Golan"

File: libraries.json [hunk 1]
```diff
@@ -3496,6 +3884,36 @@
...
-    "itemNames": ["isotopes", "protons, electrons, neutrons", "atom structure", "element, compound, mix"]
+    "itemNames": [
+      "isotopes",
+      "protons, electrons, neutrons",
+      "atom structure",
+      "element, compound, mix"
+    ]
+  },
+  {
+    "name": "Golan gophers",
```
Comment (gregoryv) [inline libraries.json:3895]: Should be "Golang gophers"

File: libraries.json [hunk 2]
```diff
@@ -3496,6 +3884,36 @@
...
+    "description": "Golang gophers we all love to spice up your drawings.",
+    "authors": [
+      {
+        "name": "Gregory Vincic",
+        "github": "https://github.com/github.com/gregoryv",
```
Comment (gregoryv) [inline libraries.json:3900]: should be "https://github.com/gregoryv" only

File: libraries.json [hunk 3]
```diff
@@ -3496,6 +3884,36 @@
...
+        "twitter": "https://twitter.com/gveet",
+        "url": "https://github.com/github.com/gregoryv"
```
Comment (gregoryv) [inline libraries.json:3902]: should be "https://github.com/gregoryv" only

**Comment by dwelle** [comment]
Oops, sorry for the delay. These are amazing! ❤️

**Pull request merged**